object_models:
  purple_cross:
    mu_r: 84.98354533597
    mu_g: 60.6422152725813
    mu_b: 84.1680430098564
    std_r: 17.3562073603737
    std_g: 21.104604511573
    std_b: 18.7768787379336
  purple_cross2:
    mu_r: 102.367348301744
    mu_g: 85.8447735151
    mu_b: 125.755065593005
    std_r: 25.5595663732303
    std_g: 27.7259334260372
    std_b: 25.4226051646474
  model_names: [purple_cross, purple_cross2]